.github/workflows/build.yaml

```diff
@@ -25,6 +25,9 @@ jobs:
 
       - run: ./gradlew build dokkaHtmlMultiModule --parallel -PredwoodNoSamples
 
+      - name: Ensure git index is clean
+        run: test -z "$(git status --porcelain | tee /dev/fd/2)"
```

Comment:
TIL https://www.youtube.com/watch?v=IJWlBfo5Oj0

Reply:
Maybe we should only check this as part of the other jobs? The check will still work since those other jobs will fail  if `kotlin-js-store` isn't up-to-date.

Could also create a new job that only runs `assemble` or something minimal then checks the git index.

Reply:
I guess it depends on when it's written. Is it during configuration? Because I'd be happy to run a no op like `tasks` on the whole project to ensure the entire configuration runs.

Reply:
Looks like it's generated by [`kotlinNpmInstall`](https://kotlinlang.org/docs/js-project-setup.html#version-locking-via-kotlin-js-store), which is pretty quick. I tried deleting the file and `kotlinNpmInstall` recreated it.

Reply:
Yeah I'm just worried because that fixes only this specific case. I'll add it and see if we run into problems like this in the future. Frankly, this is the only file I'm truly worried about anyway.